compiler/p/codegen/PPCSystemLinkage.cpp

```diff
@@ -625,6 +625,13 @@ TR::PPCSystemLinkage::mapSingleAutomatic(
 void
 TR::PPCSystemLinkage::createPrologue(TR::Instruction *cursor)
    {
+   TR::Node *firstNode = comp()->getStartTree()->getNode();
+
+   if (comp()->getOption(TR_EntryBreakPoints))
```

Comment:
Does this mean we are going to break on any method with system linkage even if we specified a filter to break on one Java method? That can be quite inconvenient.

Reply:
Option subsets will work with any OMR compiler option so you will still be able to filter methods to break on in Java.